src/components/pages/budget/BudgetList.vue

```diff
@@ -177,16 +197,35 @@
                       </span>
                     </div>
                   </td>
+                  <td class="base-salary-header text-right entry-data">
+                    {{ (personEntry.monthly_salary || 0).toLocaleString() }}
+                  </td>
+                  <td class="duration-header text-right entry-data">
+                    {{ personEntry.months_duration }}
+                  </td>
                   <td
                     :key="month"
                     class="month"
                     v-for="month in monthsBetweenProductionDates"
                   >
-                    {{
-                      personEntry.monthCosts[
+                    <input
+                      class="input-editor"
+                      type="number"
+                      :min="0"
+                      step="1"
+                      :value="getMonthCost(personEntry, month)"
+                      @change="
+                        addPersonException(
+                          personEntry,
+                          month,
+                          $event.target.value
+                        )
+                      "
+                      v-if="personEntry.monthCosts[
                         month.format('YYYY-MM')
-                      ]?.toLocaleString()
-                    }}
+                      ]"
+                    />
+                    <span v-else>&nbsp;</span>
```

Comment:
a `<span>` is mandatory?

To reduce DOM you can use `<template>`:
```suggestion
                    <template v-else>&nbsp;</template>
```